src/test/java/tests/LogicTests.java

```diff
@@ -88,6 +88,18 @@ public void replaceIssueMilestone_repoIOFails_unsuccessful() throws ExecutionExc
     }
 
     /**
+     * Tests that replaceIssueMilestone propagates exceptions when repoIO encounter exceptions
+     */
+    @Test(expected = ExecutionException.class)
```

Comment:
We can use [ExpectedException](https://github.com/HubTurbo/HubTurbo/blob/b20957b3c5f6d99901ab38c8ff21e67092c3464f/src/test/java/tests/CollaboratorServiceExTests.java#L38-L42) here. Think that's what we're supposed to standardise on.